class UserAlreadyExistsError(ValueError):
    pass

class UserNotFoundError(ValueError):
    pass